mk2driver/mk2.go

```diff
@@ -183,9 +185,11 @@ func (m *mk2Ser) updateReport() {
 
 // Checks for valid frame and chooses decoding.
 func (m *mk2Ser) handleFrame(l byte, frame []byte) {
-	logrus.Debugf("frame %#v", frame)
+	logrus.Debugf("[handleFrame] frame %#v", frame)
 	if checkChecksum(l, frame[0], frame[1:]) {
 		switch frame[0] {
+		case bootupFrameHeader:
+			m.setTarget()
```

Comment:
`setTarget()` is only executed once at startup of invertergui process. this make sure it's executed everytime Victron is started even if invertergui is running while it happens

Reply:
I think that should be all

Reply:
What do you mean by "should be all" ?

Reply:
Sorry I responded to this comment but in the wrong thread :D  https://github.com/diebietse/invertergui/issues/44#issuecomment-2243716666